backend/tests/test_participant_details.py

```diff
@@ -1,36 +1,44 @@
 from http import HTTPStatus
 
+import pytest
+
 from backend.models import Participant
 from backend.serializers.participant_serializer import ParticipantSchema
 
 
-def test_get_participant_when_logged_in(
-    client, access_token, add_participants
+@pytest.mark.parametrize("method", ["get", "delete"])
+def test_get_delete_participant_when_logged_in(
+    client, access_token, add_participants, method
 ):
-    """Test get participant details for logged in user."""
-    rv = client.get(
+    """Test get and delete participant details for logged in user."""
+    rv = getattr(client, method)(
         '/participants/1/',
         headers={'Authorization': 'Bearer {}'.format(access_token)}
     )
     response = rv.get_json()
     participant_schema = ParticipantSchema()
     assert rv.status_code == HTTPStatus.OK
-    assert response == participant_schema.dump(Participant.query.get(1))
+    if method == "get":
+        assert response == participant_schema.dump(Participant.query.get(1))
+    else:
+        assert response["message"] == "Participant deleted successfully."
 
 
-def test_get_non_existent_participant(
-    client, access_token, add_participants
+@pytest.mark.parametrize("method", ["get", "delete"])
+def test_get_delete_non_existent_participant(
+    client, access_token, add_participants, method
 ):
-    """Test get detais of non-existent participant"""
-    rv = client.get(
+    """Test get and delete non-existent participant"""
+    rv = getattr(client, method)(
         '/participants/11/',
         headers={'Authorization': 'Bearer {}'.format(access_token)}
     )
     assert rv.status_code == HTTPStatus.NOT_FOUND
 
 
-def test_get_participant_unauthorized(client, add_participants):
-    """Test get participant details for not logged in user."""
+@pytest.mark.parametrize("method", ["get", "delete"])
+def test_get_delete_participant_unauthorized(client, add_participants, method):
+    """Test get and delete participant with not logged in user."""
```

Comment:
I would remove get from all the scenarios and ensure that we have explicit one for testing `get` method. Others will be eg.: "Test delete participant when user logged-in / logged-out".

Reply:
I think it would make tests less readable, especially if we would merge #191. Following magul suggesion I introduced there logged in client fixture. So e.g. for testing get method I will have to inject two different test clients, which could be confussing.